### PR TITLE
Feature/colchecklrf

### PR DESCRIPTION
**Pull request by KdVos**

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -68,10 +68,11 @@ int main(int argc, char **argv){
 
     // The bumper class implementation uses an artificial lrf sensor
     Bumper bumper;
-    double robot_radius = 0.27; // [m]
+    double robot_radius = 0.27;
```
Comment (KdVos) [inline src/main.cpp:71]:
```suggestion
    double robot_radius = 0.21; \\ [m]
```

File: src/main.cpp [hunk 2]
```diff
@@ -68,10 +68,11 @@ int main(int argc, char **argv){
...
     bumper.setRobotRadius(robot_radius);
-    double bumperSize = 0.05; // [m]
+    double bumperSize = 0.01; // [m]
     bumper.setBumperRadius(bumperSize);
+    const bool check_collisions_by_bumper = false;
```
Comment (PetervDooren) [inline src/main.cpp:75]:
```suggestion
```

Comment (PetervDooren) [reply src/main.cpp:75]: no longer needed

File: src/moving_object.cpp [hunk 3]
```diff
@@ -42,3 +42,34 @@ geo::CompositeShapePtr makeWorldSimObject(double width, double length){
 geo::CompositeShapePtr makeWorldSimObject(MovingObject object){
     return makeWorldSimObject(object.width, object.length);
 }
+
+geo::CompositeShapePtr makeApproxRoundWorldSimObject(double radius, int nCorners){
```
Comment (PetervDooren) [inline src/moving_object.cpp:46]: Do note that geolib also has a function like this for creating cylinders

File: src/moving_object.h [hunk 4]
```diff
@@ -27,6 +27,7 @@ struct MovingObject
 
 geo::CompositeShapePtr makeWorldSimObject(double width, double height);
 geo::CompositeShapePtr makeWorldSimObject(MovingObject object);
+geo::CompositeShapePtr makeApproxRoundWorldSimObject(double radius, int nCorners);
```
Comment (PetervDooren) [inline src/moving_object.h:30]: Does not logically belong here

**Review by PetervDooren — CHANGES_REQUESTED** [review]

File: src/moving_object.cpp (hunk 3 above)
Comment (MatthijsBurgh) [reply src/moving_object.cpp:46]: https://github.com/tue-robotics/geolib2/blob/8deca1aae2581bd73cba40a91b55bf3a262d49bd/src/shapes.cpp#L11

Comment (KdVos) [reply src/moving_object.cpp:46]: Due to time limitations, changes to this part are postponed

Comment (MatthijsBurgh) [reply src/moving_object.cpp:46]: Bad excuses...

Comment (MatthijsBurgh) [reply src/moving_object.cpp:46]: Also, you should return Shape pointers, as promote them to a CompositeShape pointer only when you need to call CompositeShape only members.

**Pull request merged**